main.py

```
@@ -1,20 +1,310 @@
+        await q.message.edit_text("🔹 Main Menu", reply_markup=user_menu(is_admin(uid)))
+
+    elif q.data == "free_list":
+        kb = [[InlineKeyboardButton(f"📄 {f['name']} ({f['downloads']})", callback_data=f"get_{f['id']}")] for f in data if f["type"] == "free"]
+        await q.message.reply_text("📂 Free Files:", reply_markup=InlineKeyboardMarkup(kb) if kb else back_button(is_admin(uid)))
+
+    elif q.data.startswith("get_"):
+        fid = q.data.replace("get_", "")
+        for f in data:
+            if f["id"] == fid:
+                path = f"files/free/{f['name']}"
+                f["downloads"] += 1; save(FILES_DB, data)
```

Ruff (0.14.10)

Multiple statements on one line (semicolon)

(E702)